packages/ibm-products/src/components/EmptyStates/assets/ErrorIllustration.js

```diff
@@ -195,3 +195,5 @@ ErrorIllustration.propTypes = {
   size: PropTypes.oneOf(['lg', 'sm']),
   theme: PropTypes.oneOf(['light', 'dark']),
 };
+
+export default ErrorIllustration;
```

Comment:
The illustration components already use a named export [here](https://github.com/carbon-design-system/ibm-products/pull/4197/files#diff-f377d5706753b86522e93b86a1bf5f306c49f06683312b2788fdd39e884b6097R20). I don't think we really need to add a default export as well. But maybe there is a reason for it that I missed.

Reply:
Adding note here so we don't forget, this is required to use `React.lazy`